### DISCUSSION TITLE
Fixing TODOs

### DISCUSSION THREAD

**Discussion by deeprobin** [dotnet/runtime#69171] [Ideas]
I noticed that `git grep "TODO"` shows a lot of TODOs (which were not mentioned in an issue or something).

We should clean them up step by step in my opinion. That means evaluate if they are still relevant and then write them into issues and/or fix these TODOs.

---

My suggestion would be that each area owner records or fixes these TODOs for his area.

I think one person who only deals with TODOs is clearly too few.

**Comment by huoyaoyuan** [comment]
Does sometimes TODO mentions some long term improvement that can be a lot of work?

**Reply by deeprobin** [reply]
That may well be. That is why this should be evaluated. 
If it is something that needs a lot of work, I think we should still make it an issue instead of "TODO".
